reftests/src/tests/attributes.rs

```diff
@@ -11,3 +14,27 @@ async fn status_works(config: TestConfig) -> TestCaseResult {
 
     TestCaseResult::Success()
 }
+
+// #[derive(Serialize, Deserialize)]
+pub struct LedgerConfig {
+    // The identity of an account with tokens.
+    pub faucet: Address,
+
+    // The private key of the faucet identity.
+    pub faucet_pk: String,
+
+    // The symbol for those tokens.
+    pub symbol: Address,
+}
+
+#[derive(Serialize, Deserialize)]
+pub struct LedgerConfigJson {
+    // The identity of an account with tokens.
+    pub faucet: String,
+
+    // The private key of the faucet identity.
+    pub faucet_pk: String,
+
+    // The symbol for those tokens.
+    pub symbol: String,
+}
```

Comment:
I preferred not to do this. There is likely a shorthand for actually using serde to read into these and do everything together.